torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8777,6 +8777,14 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                                                   torch.bfloat16: 1e-1,
                                                   torch.complex64: 1e-2}),),
                    skips=(
+                       # Failing with wrong imaginary sign on at least some Windows jobs
+                       DecorateInfo(unittest.skip("Skipped!"), 'TestUnaryUfuncs', 'test_reference_numerics_normal',
```

Comment:
this is a year old issue https://github.com/pytorch/pytorch/issues/52299, it's weird that it's still not fixed, are our windows jobs using some old cuda version?